worldedit-bukkit/src/main/java/com/boydti/fawe/bukkit/adapter/mc1_16_2/BukkitAdapter_1_16_2.java

```diff
@@ -190,20 +188,25 @@ public static void sendChunk(WorldServer nmsWorld, int chunkX, int chunkZ, int m
         if (playerChunk.hasBeenLoaded()) {
             TaskManager.IMP.sync(() -> {
                 try {
-                    Set<Short>[] dirtyblocks = (Set<Short>[]) fieldDirtyBlocks.get(playerChunk);
+                    ShortSet[] dirtyblocks = (ShortSet[]) fieldDirtyBlocks.get(playerChunk);
                     if (Arrays.stream(dirtyblocks).allMatch(e -> e == null || e.isEmpty())) {
                         nmsWorld.getChunkProvider().playerChunkMap.a(playerChunk);
                     }
                     for (int i = 0; i < 16; i++) {
-                        if (dirtyblocks[i] == null) dirtyblocks[i] = (Set<Short>) shortArraySetConstructor.newInstance();
-                        for (int x = 0; x < 16; x++)
-                            for (int y = 0; y < 16; y++)
-                                for (int z = 0; z < 16; z++)
+                        if (dirtyblocks[i] == null) {
+                            dirtyblocks[i] = new ShortArraySet();
```

Comment:
This doesn't work, since WorldEdit shading overwrites the correct class for the ShortArraySet. That's why *had* to use the Reflection. https://waste.cutiepie.at/0p8rj7p0omsbnc8